frontend/src/components/workspaces/Templates/TemplatesTableRow/TemplatesTableRow.tsx

```diff
@@ -92,8 +92,7 @@ const TemplatesTableRow: FC<ITemplatesTableRowProps> = ({ ...props }) => {
       .catch(() => setCreateDisabled(false));
   };
 
-  //TODO this value will be retrieved with a query
-  const InstancesLimit = 5;
+  const InstancesLimit = data?.tenant?.status?.quota?.instances!;
```

Comment:
Not an expert of typescript, but would handle the case that value is not present (e.g., falling back to a default), instead of asserting that it is always present. @QcFe WDYT?